packages/circuit-ui/components/CalendarTag/CalendarTag.js

```diff
@@ -37,18 +36,24 @@ class CalendarTag extends Component {
      */
     onDatesRangeChange: PropTypes.func.isRequired,
     /**
-     * Additional data that is dispatched with the tracking event.
+     * @deprecated
+     *
+     * Use an `onClick` handler to dispatch user interaction events instead.
      */
     tracking: PropTypes.shape({
       label: PropTypes.string.isRequired,
       component: PropTypes.string,
       customParameters: PropTypes.object,
     }),
+    /**
+     * Function that's called when the date tag is clicked.
+     */
+    onClick: PropTypes.func,
```

Comment:
Was about to suggest:

```suggestion
    onClick?: PropTypes.func,
```


...and then I realized this was not TypeScript 😄